tests/markers.py

```diff
@@ -13,6 +13,7 @@
 parametrize = mark.parametrize
 test_case = pytestrail.case
 xfail = mark.xfail
+skip = mark.skip
```

Comment:
Where are you using the skip marker? I'm not seeing it on any of the tests.

Reply:
I removed the function which used it. Removed the marker skip.